src/translations/client.csv

```diff
@@ -2602,156 +2608,157 @@ v2.event.tennis-club-membership.action.delete.label,This is shown as the action
+v2.events.history.status,Events status history,"{status, select, CREATE {Draft} VALIDATE {Validated} DRAFT {Draft} DECLARE {Declared} REGISTER {Registered} PRINT_CERTIFICATE {Print certificate} other {Unknown}}","{status, select, CREATE {Brouillon} VALIDATE {Déclaration envoyée pour approbation} DRAFT {Brouillon} DECLARE {Déclaration initiée} REGISTER {Enregistré} PRINT_CERTIFICATE {Certifié} other {Inconnu}}"
+v2.exitModal.cancel, Modal cancel button,Cancel,Annuler
+v2.exitModal.exitWithoutSaving,exitmodal without saving label,Exit without saving changes?,Quitter sans enregistrer les modifications?
+v2.exitModal.exitWithoutSavingDescription, exitmodal without saving  description,You have unsaved changes on your declaration form. Are you sure you want to exit without saving?,Vous avez des modifications non enregistrées sur votre formulaire de déclaration. Voulez-vous vraiment quitter sans enregistrer ?
+v2.field.address.country.label,Label for country in address,Country,Pays
+v2.field.address.district.label,Label for district in address,District,Arrondissement
+v2.field.address.label.rural,Label for rural area,Rural,Zone rurale
+v2.field.address.label.urban,Label for urban area,Urban,Urbain
+v2.field.address.number.label,Label for address number,Number,Numéro
+v2.field.address.province.label,Label for province in address,Province,Province
+v2.field.address.residentialArea.label,Label for residential area in address,Residential Area,Zone résidentielle
+v2.field.address.street.label,Label for street in address,Street,Rue
+v2.field.address.town.label,Label for town in address,Town,Ville
+v2.field.address.urbanOrRural.label,Label for selecting between urban or rural,Urban or Rural,Urbain ou rural
+v2.field.address.village.label,Label for village in address,Village,Village
+v2.field.address.zipCode.label,Label for ZIP code in address,ZIP Code,Code postal
+v2.form.birth.child.title,Form section title for Child,Child's details,information de l'enfant
+v2.form.field.label.No,Label for form field radio option No,No,Non
+v2.form.field.label.Yes,Label for form field radio option Yes,Yes,Oui
+v2.form.field.label.attendantAtBirthLayperson,Label for layperson attendant,Layperson,personne lamda
+v2.form.field.label.attendantAtBirthMidwife,Label for midwife attendant,Midwife,Sage-femme
+v2.form.field.label.attendantAtBirthNone,Label for no attendant,None,Aucun
+v2.form.field.label.attendantAtBirthNurse,Label for nurse attendant,Nurse,Infirmière
+v2.form.field.label.attendantAtBirthOtherParamedicalPersonnel,Label for other paramedical personnel,Other paramedical personnel,Autre personnel paramédical
+v2.form.field.label.attendantAtBirthPhysician,Label for physician attendant,Physician,Médecin
+v2.form.field.label.attendantAtBirthTraditionalBirthAttendant,Label for traditional birth attendant,Traditional birth attendant,Accoucheur (se) Traditionnel(le)
```

Comment:
This should be "Accoucheur(se) Traditionnel(le)"

// could you please remove the space?